index.js

```diff
@@ -33,10 +35,12 @@ function openConfig (cb) {
     return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
   }
 
-  tryOpenConfig(path.join(userHome, configfile), function (err, fd) {
-    if (err) return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
-    return cb(null, fd);
-  });
+  fs.mkdir(configPath, function () {
```

Comment:
Even if it isn't worth the added dependency (not sure if so), checking the error might still be worth it.